feat(move-to-api-v6) #WPB-11222 Update and adapt to Wire backend API … (#12)
wireapp/helium: README.md

# Helium

Wire Http Client written in Java

Helium implements WireAPI defined in Xenon repository,
making HTTP Rest calls to the Wire backend.

Current backend API targeted version is v6, set host and version with environmental variable "WIRE_API_HOST".

> Not all the APIs available are implemented, but Helium should help with the most common task of an SDK to send/receive messages and manage the users' data.

Reviewer: ```suggestion
Not all the APIs available are implemented, but Helium should help with the most common task of an SDK to send/receive messages and manage the user's data.
```

Author: Done

    <dependency>
        <groupId>com.wire</groupId>
        <artifactId>helium</artifactId>
        <version>x.y.z</version>
    </dependency>
</dependencies>
```

> Create a `LoginClient`, passing a Java WS RS http client previously created, then obtain the users' token

Reviewer: ```suggestion
Create a `LoginClient`, passing a Java WS RS http client previously created, then obtain the user's token
```

Author: Done

> Create a `LoginClient`, passing a Java WS RS http client previously created, then obtain the users' token

Reviewer: What does `Java WS RS` stands for? maybe we could write the full meaning so its easier for other outside (me included 😅 )

Author: I'm with you here, unfortunately it's not me being lazy, that's the actual specification name. Back in the day the Java community really loved abbreviations. I changed the comment a bit and also wrote Jersey client as an example to be more precise

> with the login call. Then, create a `API` instance with the same http-client and the users' token.

Reviewer: ```suggestion
with the login call. Then, create an `API` instance with the same http-client and the user's token.
```

Author: Done
